doc/etc/config.ini

```diff
@@ -36,10 +36,17 @@ path=/srv/dlsnode/dlsnode.pid
 ;
 ; write_flush_ms        millisecond interval at which to flush the buffers of
```

Comment:
not related to this PR, but the documentation for this configuration variable is missing in the definition of the variable  in https://github.com/sociomantic-tsunami/dlsnode/blob/master/src/dlsnode/app/config/PerformanceConfig.d#L25